### PR TITLE
Solves #12(Added badges to Readme)

### PR DESCRIPTION
**Pull request by Archies11**
I have added some badges.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -4,6 +4,15 @@
 
 > Made with :heart: for Windows 💻  users, who cannot use `touch` command to create files :+1:
 
+## Badges
```
Comment (debck) [inline README.md]: It is not needed. Just delete it.

**Comment by debck** [comment]
You can use https://shields.io/#/ or https://forthebadge.com/ for creating the badges

**Comment by Archies11** [comment]
Should I change the style of the badges too?

**Comment by debck** [comment]
No need. Just add those.

**Comment by Archies11** [comment]
I have made the changes.

**Comment by debck** [comment]
Thats good. I answered your question in  slack . Can you please see it and  add required badges too

**Comment by Archies11** [comment]
I have added some more badges.

**Comment by Archies11** [comment]
I have made the required changes.

**Comment by debck** [comment]
Now it fine. I will add the Codacy status link later.

**Comment by debck** [comment]
@Archies11  always remember to mention the issue no. in the PR as :` Fixes #<issue number>`

**Comment by Archies11** [comment]
Ok, I will surely keep that in mind. Thank you

**Pull request merged**